tests/integration/network_test.go

```diff
@@ -0,0 +1,106 @@
+package integration_test
+
+import (
+	"fmt"
+	"testing"
+
+	tmcli "github.com/cometbft/cometbft/libs/cli"
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"google.golang.org/grpc/status"
+
+	"github.com/skip-mev/block-sdk/testutils/networksuite"
+	auctioncli "github.com/skip-mev/block-sdk/x/auction/client/cli"
+	auctiontypes "github.com/skip-mev/block-sdk/x/auction/types"
+	blocksdkcli "github.com/skip-mev/block-sdk/x/blocksdk/client/cli"
+	blocksdktypes "github.com/skip-mev/block-sdk/x/blocksdk/types"
+)
+
+// NetworkTestSuite is a test suite for network integration tests.
+type NetworkTestSuite struct {
+	networksuite.NetworkTestSuite
+}
+
+// TestQueryTestSuite runs test of network integration tests.
+func TestNetworkTestSuite(t *testing.T) {
+	suite.Run(t, new(NetworkTestSuite))
```

Comment:
question: is there a way to get an API endpoint here for REST / GRPC queries, or can we only exec via CLI?

Reply:
s.Network.Validators each have endpoints you can use !